### ISSUE TITLE
[GFX9918] Produce batch #1 of GFX9918 Rev2

### ISSUE THREAD

**Issue by apoloval** [apoloval/artemisa#52]
After #46, we are ready to produce the first batch of GFX9918. 

We will start with 5 units. 

Pre-production
- [x] Ensure the PCB is updated and revision number is set
- [x] Generate the Gerber files and submit an order to the factory

Ensure stock of BOM:
- [x] 1x TMS9918
- [x] 1x HM62256ALP-10
- [x] 1x 74HCT04
- [x] 3x 74HCT574
- [x] 1x 74HC32
- [x] 1x Crystal 10.7386Mhz
- [x] 2x Disc capacitor 27pF
- [x] 7x Disc capacitor 100nF
- [x] 1x Radial capacitor 10uF
- [x] 1x Axial resistor 470
- [x] 1x Socket DIP 40 600"
- [x] 1x Socket DIP 28 600
- [x] 1x RCA yellow
- [x] 1x RCA white
- [x] 1x RCA red
- [x] 1x IDC 50p PCB connector male

Post-production
- [x] Prepare kits
- [ ] Assemble and solder the components
- [ ] Test the units using an Arduino board

**Comment by apoloval** [comment]
GFX9918 Rev2 [released](https://github.com/apoloval/artemisa/releases/tag/gfx9918-rev2)!

**Comment by apoloval** [comment]
Order submitted to the manufacturer. Waiting for audit.

**Comment by apoloval** [comment]
Audit passed and payment done. Waiting for manufacturing.

**Comment by apoloval** [comment]
Boards arrived yesterday. Ready to soldering and packaging!

**Comment by apoloval** [comment]
Kits prepared. Issues:
- 74HCT574: no stocks. 20 units ordered
- 74HCT04: low on stocks. 20 units ordered. 
- 27pf ceramic capacitor: low on stocks. 100 units ordered. 
- 100nF ceramic capacitor: low on stocks. 500 units ordered. 
- 10uF electrolytic capacitor: low on stocks. 60 units ordered.

**Comment by apoloval** [comment]
Waiting for 74HC574 to complete the batch. Also for some ESD bags and protection bags that fit better in the box.

**Comment by apoloval** [comment]
Already done.
